### PR TITLE
Update Set-SPOTenant.md

### PR DESCRIPTION
**Pull request by nishantbhatt09**
Add MediaAutomaticTranscriptionFeature tenant setting

### CODE REVIEW

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 1]
```diff
@@ -1108,6 +1109,26 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -MediaTranscriptionAutomaticFeatures
+
+When the feature is enabled, videos can have transcript generated automatically on upload. The policy is default on. If a tenant admin decides to disable the feature, he can do so by disabling the policy at tenant level. This feature can not be enabled/disabled at site level.
+Possible values:
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:1115]:
```suggestion
When the feature is enabled, videos can have transcripts generated automatically on upload. The policy is default on. If a tenant admin decides to disable the feature, he can do so by disabling the policy at tenant level. This feature can not be enabled or disabled at site level.

Possible values are:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @nishantbhatt09, kindly commit the suggested change. Many thanks.

**Comment by yogkumgit** [comment]
@nishantbhatt09 Please implement the suggested changes. Thanks

**Review by get-itips — APPROVED** [review]

**Review by dariomws — APPROVED** [review]

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md (hunk 1 above)
Comment (nishantbhatt09) [reply sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md]: Addressed your comment

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**